packages/webview-common/src/components/button.tsx

```diff
@@ -10,20 +10,19 @@ export const PrimaryButton = ({
   return (
     <button
       className={cn(
-        'relative inline-flex h-12 overflow-hidden rounded-full p-[2px] focus:outline-none focus:ring-2 focus:ring-slate-400 focus:ring-offset-2 focus:ring-offset-slate-50',
+        'relative inline-flex overflow-hidden rounded-full bg-gray-800 p-[2px] focus:outline-none focus:ring-2 focus:ring-slate-400 focus:ring-offset-2 focus:ring-offset-slate-50',
```

Comment:
This component is only used for the Guess button so it's okay to change to match the specific case